java/src/main/java/ola/OlaClient.java

```diff
@@ -114,14 +117,23 @@ public PluginListReply getPlugins() {
     }
 
 
+    /**
+     * Reload the plugins.
+     *
+     * @return Acknowledgement.
+     */
+    public Ack reloadPlugins() {
+        return (Ack) callRpcMethod("ReloadPlugins", PluginReloadRequest.newBuilder().build());
```

Comment:
I think the behaviour of patchPort is more useful:
https://github.com/OpenLightingProject/ola/blob/c4adb5034feddeaea32cb4554ef00837dbbcbec9/java/src/main/java/ola/OlaClient.java#L240
https://github.com/OpenLightingProject/ola/blob/c4adb5034feddeaea32cb4554ef00837dbbcbec9/java/src/main/java/ola/OlaClient.java#L252

Reply:
@peternewman Sorry, not sure, what you mean. With patchPort, no reloadPlugins would be required? I thought I have to use it after activating/deactivating plugins in my client, but seems like it's not required. I can remove it, if you think it's not necessary.

Reply:
@peternewman Ah, you mean the return value? Changed it with commit b5be0b94de51153921ab8c051f000487f9d3db9f